.asf.yaml

```diff
@@ -28,3 +28,8 @@ github:
     main:
       required_status_checks:
         strict: true
+        contexts:
+          - build-license-eye
```

Comment:
This is not right

Reply:
What should put here? I thought it  should be task name.